server/config.go

```diff
@@ -161,6 +165,17 @@ func (cfg *Config) VerificationCfg() *verify.Config {
 
 // ReadConfig ... parses the Config from the provided flags or environment variables.
 func ReadConfig(ctx *cli.Context) Config {
+	signerPrivateKeyHex := func() string {
+		if key := ctx.String(SignerPrivateKeyHexFlagName); key != "" {
+			return key
+		}
+		randomBytes := make([]byte, 32)
+		if _, err := rand.Read(randomBytes); err != nil {
+			return ""
+		}
+		return hex.EncodeToString(randomBytes)
+	}()
```

Comment:
```suggestion
	signerPrivateKeyHex := ctx.String(SignerPrivateKeyHexFlagName)
	if signerPrivateKeyHex == "" {
		randomBytes := make([]byte, 32)
		if _, err := rand.Read(randomBytes); err != nil {
			return ""
		}
		signerPrivateKeyHex = hex.EncodeToString(randomBytes)
	}
```
Can we do something simpler like this? Really don't like inline functions when not needed.

Reply:
Also not sure about returning "" when rand fails... maybe panic was right. Not super sure what the intended behavior is here so I'll leave that to you.